Provide API ID in the API specification

This was breaking the app:

```
clojure.lang.ExceptionInfo: Error in component :api in system com.stuartsierra.component.SystemMap calling #'com.stuartsierra.component/start
	at clojure.core$ex_info.invokeStatic(core.clj:4617) ~[even.jar:?]
	at clojure.core$ex_info.invoke(core.clj:4617) ~[even.jar:?]
	at com.stuartsierra.component$try_action.invokeStatic(component.cljc:119) ~[even.jar:?]
	at com.stuartsierra.component$try_action.invoke(component.cljc:116) ~[even.jar:?]
	at com.stuartsierra.component$update_system$fn__1089.invoke(component.cljc:139) ~[even.jar:?]
	at clojure.lang.ArraySeq.reduce(ArraySeq.java:114) ~[even.jar:?]
	at clojure.core$reduce.invokeStatic(core.clj:6544) ~[even.jar:?]
	at clojure.core$reduce.invoke(core.clj:6527) ~[even.jar:?]
	at com.stuartsierra.component$update_system.invokeStatic(component.cljc:135) ~[even.jar:?]
	at com.stuartsierra.component$update_system.doInvoke(component.cljc:129) ~[even.jar:?]
	at clojure.lang.RestFn.invoke(RestFn.java:445) ~[even.jar:?]
	at com.stuartsierra.component$start_system.invokeStatic(component.cljc:163) ~[even.jar:?]
	at com.stuartsierra.component$start_system.invoke(component.cljc:155) ~[even.jar:?]
	at com.stuartsierra.component$start_system.invokeStatic(component.cljc:161) ~[even.jar:?]
	at com.stuartsierra.component$start_system.invoke(component.cljc:155) ~[even.jar:?]
	at com.stuartsierra.component.SystemMap.start(component.cljc:178) ~[even.jar:?]
	at org.zalando.stups.friboo.system$run.invokeStatic(system.clj:110) ~[even.jar:?]
	at org.zalando.stups.friboo.system$run.invoke(system.clj:92) ~[even.jar:?]
	at org.zalando.stups.even.core$run.invokeStatic(core.clj:46) ~[even.jar:?]
	at org.zalando.stups.even.core$run.invoke(core.clj:34) ~[even.jar:?]
	at org.zalando.stups.even.core$_main.invokeStatic(core.clj:52) [even.jar:?]
	at org.zalando.stups.even.core$_main.doInvoke(core.clj:48) [even.jar:?]
	at clojure.lang.RestFn.invoke(RestFn.java:397) [even.jar:?]
	at clojure.lang.AFn.applyToHelper(AFn.java:152) [even.jar:?]
	at clojure.lang.RestFn.applyTo(RestFn.java:132) [even.jar:?]
	at org.zalando.stups.even.core.main(Unknown Source) [even.jar:?]
Caused by: clojure.lang.ExceptionInfo: Value does not match schema: {"info" {"x-api-id" disallowed-key}}
	at schema.core$validator$fn__2428.invoke(core.clj:155) ~[even.jar:?]
	at schema.core$validate.invokeStatic(core.clj:164) ~[even.jar:?]
	at schema.core$validate.invoke(core.clj:159) ~[even.jar:?]
	at io.sarnowski.swagger1st.context$create_context.invokeStatic(context.clj:37) ~[even.jar:?]
	at io.sarnowski.swagger1st.context$create_context.invoke(context.clj:32) ~[e

--- a/resources/api/even-api.yaml
+++ b/resources/api/even-api.yaml
@@ -3,7 +3,6 @@ swagger: '2.0'
 info:
   version: "0.1"
   title: even
-  x-api-id: c579a373-82a3-4b33-ad26-af1de73b907d
   description: SSH access granting service
 
 basePath: /

